src/controllers/users.js

```diff
@@ -65,21 +65,22 @@ async function checkSaltNonce(saltNonce, address, safeAddress) {
 }
 
 async function checkIfExists(username, safeAddress) {
+  const caseInsensitiveUsername = sequelize.where(
```

Comment:
I wish we ha typescript haha
I was reading the documentation of sequalize but I was for a long time confused about what kind of object this is. I think the name confused me too. This is a definition of what is a match and not the caseInsensitiveUsername, correct? I would expect a variable named like this to be `username` for `UserName` or something. Perhaps we can find a more clear name? Or maybe it is just unclear to me because I haven't used sequalize before.

Reply:
> This is a definition of what is a match and not the caseInsensitiveUsername, correct? 

This is a DSL which generates an SQL query aka "SELECT * FROM .. WHERE username = '<value>'" where `<value>` is automatically lowercased.

Reply:
Do you have an idea for a better naming @louilinn?

Reply:
Maybe `caseInsensitiveUsernameCondition`? :sweat_smile: